src/stacks-hierarchy/Root_PurchaseOverviewScreen/Root_PurchaseOverviewScreen.tsx

```diff
@@ -91,6 +92,9 @@ export const Root_PurchaseOverviewScreen: React.FC<Props> = ({
     requiresTokenOnMobile,
   } = params.fareProductTypeConfig.configuration;
 
+  const fareProductOnBehalfOfEnabled =
+    params.fareProductTypeConfig.onBehalfOfEnabled;
```

Comment:
It's a similar situation with the `isCollectionOfAccesses` 🙈 , I think we can just go with the current setting and change it later on another PR.

Reply:
You're maybe right, but I feel like `isCollectionOfAccesses` is slightly different: it describes something about the product itself, while `onBehalfOfEnabled` is some config related to the purchase process. 

I usually wouldn't mind, and agree that it can be changed later, but these configs have to be backwards compatible, so changing things like this might mean we need to deprecate the old configs, and go over to a `fareProductTypeConfigV2`. So even though it's annoying, it's easier to change now, rather than later. 

If you do, I'll try to do quick reviews so it can be faster than the previous chain of PRs 😅 

Do you have an opinion @gorandalum?

Reply:
I see, then let's wait for @gorandalum opinion and I'll do the action :D

Reply:
Yeah, I think this is purchase configuration, and should go into `configuration`.

Reply:
Ok, will make some changes first on the firestore config 👍